spec/requests/ping_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe 'Pings', type: :request do
+  describe 'GET /ping' do
+    context 'when the correct environment variables have been set during the deploy process' do
+      before do
+        allow(ENV).to receive(:fetch).with('APP_BUILD_DATE').and_return('2017-04-06T14:47:13+0000')
```

Comment:
**[not a blocker]** 💡 I've come across this [gem](https://github.com/thoughtbot/climate_control) that achieves the same thing in a slightly cleaner and more [controlled](https://github.com/thoughtbot/climate_control#why-use-climate-control) away.

For this specific case it can be used with a _RSpec around_ block